ERCS/erc-7208.md

```diff
@@ -271,6 +271,8 @@ We present an **educational example** implementation showcasing two types of tok
 **This example has not been audited and should not be used in production environments.**
 
 
+See [contracts](../assets/erc-7208/contracts/README.md)
```

Comment:
Because of a quirk of our rendering system, this needs to be:

```suggestion
See [contracts](../assets/eip-7208/contracts/README.md)
```

Reply:
Done! Thank you!